contracts/examples/adder/interactor/tests/basic_interactor_cs_test.rs

```diff
@@ -33,3 +34,30 @@ async fn simulator_upgrade_test() {
     let sum = basic_interact.get_sum().await;
     assert_eq!(sum, 7u32.into());
 }
+
+#[tokio::test]
+#[cfg_attr(not(feature = "chain-simulator-tests"), ignore)]
+async fn set_state_cs_test() {
+    let account_address = test_wallets::mike();
+
+    let real_chain_interact = AdderInteract::new(Config::load_config()).await;
+    let simulator_interact = AdderInteract::new(Config::chain_simulator_config()).await;
+
+    let account = real_chain_interact
```

Comment:
We should save the account somewhere.
These are separate actions.
The most common scenario will be:
1. We download the state once, save it to some file, or in code. Mandos SetState is already implemented, to be decided if we need something else.
2. We read from file, set the state to chain simulator and run our test.

It is also possible to simulate starting from real chain, but this is not reliable as a test, since the mainnet state might be compromised. So we should keep this function as well, but not as an integration test.